src/frontend/js/widgets/Dashboard/components/DashboardAvatar/index.spec.tsx

```diff
@@ -1,15 +1,16 @@
-import { render, screen } from '@testing-library/react';
+import { screen } from '@testing-library/react';
 import { JoanieFileFactory } from 'utils/test/factories/joanie';
+import { render } from 'utils/test/render';
 import { DashboardAvatar, DashboardAvatarVariantEnum } from '.';
 
 describe('<DashboardAvatar/>', () => {
   it('should work with empty title', () => {
-    render(<DashboardAvatar title="" />);
+    render(<DashboardAvatar title="" />, { wrapper: null });
```

Comment:
Maybe we could make options optional ?

Reply:
`options` is optional.
By default it's :
```ts
const toto = {
  wrapper: JoanieAppWrapper,
  queryOptions: {
    client: createTestQueryClient({ user: true })
  }
  intlOption: {
    locale: 'en',
  }
}
```

Reply:
Oh right understood